### PR TITLE
Add create and get all customers

### PR DESCRIPTION
**Pull request by ValentasK**

### CODE REVIEW

File: API.V1/Handlers/GetCustomerByIdHandler.cs [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+{
+    public class GetCustomerByIdHandler : IRequestHandler<GetCustomerByIdQuery, CustomerResponse>
+    {
+        public async Task<CustomerResponse> Handle(GetCustomerByIdQuery request, CancellationToken cancellationToken)
+        {
+            var cs = @"Server=(localdb)\MSSQLLocalDB;Database=DapperDB;Trusted_Connection=True;";
+            using var con = new SqlConnection(cs);
+            con.Open();
+
+            string sql = "select * from [DapperDB].[dbo].[Person] as P\r\n  left join [DapperDB].[dbo].[Phone] as ph on p.CellPhoneId = ph.Id\t\r\n  where p.Id = @Id;";
```
Comment (KestasMil) [inline API.V1/Handlers/GetCustomerByIdHandler.cs:17]: Kam cia tie \r\n sql reikalingi?

File: ConsoleApp_For_Katas/Program.cs [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        {
+
+            //Consonants are all letters used to write English excluding the vowels a, e, i, o, u.
+
+            // [TestCase("h^$&^#$&^elLo world", ExpectedResult = 7)]
+
+            string aaa = "helLo world";    // 7
+            int count = 0;
+
+            var g = aaa.ToLower();
```
Comment (KestasMil) [inline ConsoleApp_For_Katas/Program.cs:25]: Kas cia per variable name 'aaa' ???

File: The_Super_App/Program.cs [hunk 3]
```diff
@@ -5,12 +5,37 @@
 
 Console.WriteLine("Hello, The Super App");
 
-var cs = @"Server=(localdb)\MSSQLLocalDB;Database=DapperDB;Trusted_Connection=True;;";
+var cs = @"Server=(localdb)\MSSQLLocalDB;Database=DapperDB;Trusted_Connection=True;";
```
Comment (KestasMil) [inline The_Super_App/Program.cs:8]: Connection string -> config file!

File: API.V1/Handlers/GetAllOrdersHandller.cs [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    public class GetAllCustomersHandller : IRequestHandler<GetAllCustomersQuery, List<CustomerResponse>>
+    {
+        public async Task<List<CustomerResponse>> Handle(GetAllCustomersQuery request, CancellationToken cancellationToken)
+        {
+
+            var cs = @"Server=(localdb)\MSSQLLocalDB;Database=DapperDB;Trusted_Connection=True;";
+            using var con = new SqlConnection(cs);
+            con.Open();
+
+            string sql = "select * from [DapperDB].[dbo].[Person] as P\r\n  left join [DapperDB].[dbo].[Phone] as ph on p.CellPhoneId = ph.Id\t";
```
Comment (KestasMil) [inline API.V1/Handlers/GetAllOrdersHandller.cs:19]: \r\n ???

**Review by KestasMil — CHANGES_REQUESTED** [review]

File: ConsoleApp_For_Katas/Program.cs (hunk 2 above)
Comment (ValentasK) [reply ConsoleApp_For_Katas/Program.cs]: ConsoleApp_For_Katas vapsie cia net neturejo buti. Istrintas

File: API.V1/Handlers/GetAllOrdersHandller.cs (hunk 4 above)
Comment (ValentasK) [reply API.V1/Handlers/GetAllOrdersHandller.cs]: Fixed.

File: API.V1/Handlers/GetCustomerByIdHandler.cs (hunk 1 above)
Comment (ValentasK) [reply API.V1/Handlers/GetCustomerByIdHandler.cs]: Fixed

File: The_Super_App/Program.cs (hunk 3 above)
Comment (ValentasK) [reply The_Super_App/Program.cs:8]: Ble sita reiks rytoj sutvarkyti

Comment (ValentasK) [reply The_Super_App/Program.cs]: Fixed

File: API.V1/Program.cs [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            var builder = WebApplication.CreateBuilder(args);
+
+            // Add services to the container.
+
+            builder.Services.AddControllers();
+            // Learn more about configuring Swagger/OpenAPI at https://aka.ms/aspnetcore/swashbuckle
```
Comment (KestasMil) [inline API.V1/Program.cs:17]: Yes

**Review by KestasMil — CHANGES_REQUESTED** [review]

File: API.V1/Program.cs (hunk 5 above)
Comment (ValentasK) [reply API.V1/Program.cs:17]: Not sure what is the question

File: API.V1/Handlers/GetAllOrdersHandller.cs [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+    public class GetAllCustomersHandller : IRequestHandler<GetAllCustomersQuery, List<CustomerResponse>>
+    {
+         IConfiguration _configuration;
+        public GetAllCustomersHandller(IConfiguration configuration)
+        {
+            _configuration = configuration;
+        }
+        public async Task<List<CustomerResponse>> Handle(GetAllCustomersQuery request, CancellationToken cancellationToken)
+        {
+            string? connectionString = _configuration.GetConnectionString("myDb1");
```
[the remark this answers is not in the document]
Comment (ValentasK) [reply API.V1/Handlers/GetAllOrdersHandller.cs]: Done

**Review by KestasMil — APPROVED** [review]

**Pull request merged**